### DISCUSSION TITLE
Alternative naming for federal subjects.

### DISCUSSION THREAD

**Discussion by RVA2869** [govdirectory/website#412] [Ideas]
![afbeelding](https://github.com/user-attachments/assets/c6c5d4d7-b1c6-4d6d-b793-4b40c9c6cfab)
What if we use the type of "sub country/federal subject"?
For the case of Scotland it should be something like `constituent country of the United Kingdom`
Germany -> federated state of Germany
USA -> U.S. state
...
You get the gist. 

But make sure that more than 1 type can be added. So that we don't have [Guam](https://www.wikidata.org/wiki/Q16635) under the US states for example.

**Comment by Ainali** [comment]
I don't understand fully. These pages are generated, so if we would be doing it differently on each page, then we need a property that has this. Is there a good one to use? 
And would Bermuda (#408) fit in that?
Or were you suggesting "sub country/federal subject" as the string instead of "Federal subject"?

**Reply by RVA2869** [reply]
> And would Bermuda (https://github.com/govdirectory/website/issues/408) fit in that?

Bermuda would fall under the "category" of British Overseas Territories.

> Or were you suggesting "sub country/federal subject" as the string instead of "Federal subject"?

The type is the name (instead of Federal subjects)

**Reply by Ainali** [reply]
> Bermuda would fall under the "category" of British Overseas Territories.

And where would British Overseas Territories fit in?

**Reply by Ainali** [reply]
> The type is the name (instead of Federal subjects)

You mean "instance of (P31)"? So, for example, then Scotland is Q3336843. I guess that might work if we define it with one extra parameter on the "hosting country" in the countries query.

**Comment by Ainali** [comment]
Issue now at #569

**Comment by Abbe98** [comment]
Lower-tier government?
Subsidiary government?
Regional government?

I think we should avoid the term "national" in this context.

**Reply by Ainali** [reply]
> I think we should avoid the term "national" in this context.

I understand that long-term, when we may have supranational entities like the EU. But as short-term fix for the two ones we have, I think it can work (as some Americans is even hesitating to promote the website now).

**Reply by Abbe98** [reply]
I see, my problem in this moment is with UK/Scotland as sub-national reads very odd in that context.

**Comment by Ainali** [comment]
How about just "Constituents" or "Constituent parts"? I was inspired by reading on [English Wiktionary about constituent country](https://en.wiktionary.org/wiki/constituent_country).

**Comment by Ainali** [comment]
During the collab call today we opted to start with "Parts of {{country}}".
